test/functional/rpc_getmininginfo.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the getmininginfo API."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+from decimal import Decimal
```

Comment:
supernit: place imports in PEP-8 ordering (std library first, then project imports)